modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/domain/statechannel/StateChannelValidator.scala

```diff
@@ -52,6 +54,13 @@ object StateChannelValidator {
       case (address, Right((_, info))) => address -> info.lastMessages.toList.flatMap(_.values.map(_.address).toList).toSet
     }
 
+  def getFeeAddresses(
+    lastCurrencySnapshots: SortedMap[Address, Either[Signed[CurrencySnapshot], (Signed[CurrencyIncrementalSnapshot], CurrencySnapshotInfo)]]
```

Comment:
I'm not sure we can fix this quite yet, but huge arguments like this are indicative of an issue where we're passing large maps around everywhere instead of having some kind of wrapper type. A getter like this should really belong on some kind of bigger object, rather than being it's own isolated instance that takes a huge argument and then does a small function on it.